src/Content/Text/Plaintext.php

```diff
@@ -23,9 +23,12 @@
 
 use Friendica\Core\Protocol;
 use Friendica\DI;
+use Friendica\Util\Network;
 
 class Plaintext
 {
+	const URL_LENGTH = 23;
```

Comment:
Thank you for adding a constant, can you please add documentation to it?

Reply:
Done